package/toltec-completion/package

```diff
@@ -5,8 +5,8 @@
 pkgnames=(toltec-completion)
 pkgdesc="Expands bash-completion with functions for toltec-specific commands"
 url=https://github.com/toltec-dev/toltec
-pkgver=0.3.0-1
-timestamp=2021-07-29T22:55Z
+pkgver=0.3.1-1
```

Comment:
Since you are just removing a comment, I would probably just increment the package release version, so `0.3.0-2`

Reply:
I had already pondered this decision and am somewhat against it, since strictly speaking "the source code" was updated/patched here and not the packaging. But I can also understand your point since the functionality of the software doesn't change with this.

Feel free to decide on your own whether to apply this suggestion.

Reply:
Technically it's up to you since you maintain this package :P